sdks/python/apache_beam/yaml/yaml_transform.py

```diff
@@ -771,6 +771,9 @@ def preprocess_windowing(spec):
         'transforms': [modified_spec] + windowing_transforms,
         'input': spec['input'],
         'output': modified_spec['__uuid__'],
+        'config': {
+            'error_handling': spec.get('config').get('error_handling', None)
+        } or {},
```

Comment:
I don't think the `or {}` will ever trigger, as this will always result in a key with a single dictionary entry. However, setting letting error_handling=None should be fine.

Reply:
Good point